### PR TITLE
fix(cmd): tty should auto disable color

### PR DESCRIPTION
**Pull request by Arqu**
Sprint board didn't update so I got started on https://github.com/qri-io/qri/issues/955
Wound up on the https://github.com/qri-io/ioes/commit/28794e5a4c834251ed7e0fde143fd3c99bb58db3 commit.
Once we cut the release on `ioes` I'll update the deps here and we can merge.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -50,7 +50,7 @@ require (
 	github.com/qri-io/dataset v0.1.5-0.20200324184139-108a69072ede
 	github.com/qri-io/deepdiff v0.1.1-0.20200305020550-8173efebcaa1
 	github.com/qri-io/doggos v0.1.0
-	github.com/qri-io/ioes v0.1.0
+	github.com/qri-io/ioes v0.1.1-0.20200407033451-28794e5a4c83
```
Comment (b5) [inline go.mod:53]: as requested, I've now published v0.1.1

File: cmd/qri.go [hunk 2]
```diff
@@ -32,6 +32,10 @@ https://github.com/qri-io/qri/issues`,
 			if err == nil && noColor {
 				setNoColor(noColor)
 			}
+			// if it's a tty terminal disable color
+			if ioStreams.IsTerminal() {
```
Comment (b5) [inline cmd/qri.go:36]: I think this is having the opposite effect of what's intended? Do we want `IsCygwinTerminal` instead? I'm not an expert here

File: cmd/qri.go [hunk 3]
```diff
@@ -32,6 +32,10 @@ https://github.com/qri-io/qri/issues`,
...
+				setNoColor(noColor)
```
Comment (b5) [inline cmd/qri.go:37]: What happens when the user supplies `qri --no-color=false`?

File: cmd/qri.go (hunk 2 above)
Comment (Arqu) [reply cmd/qri.go:36]: Still not sure on either option based on this: https://github.com/qri-io/qri/pull/1262#pullrequestreview-389020225

File: cmd/qri.go (hunk 3 above)
Comment (Arqu) [reply cmd/qri.go:37]: Good point, though I don't see how both of it should work at the same time. I guess leaving the option to the user makes sense. Though I will check if the flag was changed and not just default-ed.

**Comment by Arqu** [comment]
Still not completely sure, want to address this during the day. I'll set up a Win environment and just play with it to get some field experience on it.

**Comment by Arqu** [comment]
There was a lot of testing on this, in the end, the most sensible approach turned out to be:
- disable color on windows by default
- let users override it with `--no-color=false`

The reasoning being it's just too much effort to track and maintain different combinations of os/color support as they all revolve around different hacks to test for it. 

From some manual testing I found this (on a Win10 machine):

cmd
- IsTerminal = true
- IsCygwinTerminal = false
- Color does not work

PowerShell
- IsTerminal = true
- IsCygwinTerminal = false
- Color does not work

MSYS2
- IsTerminal = false
- IsCygwinTerminal = true
- Color does work

Bash
- IsTerminal = true
- IsCygwinTerminal = false
- Color does work

Cygwin
- IsTerminal = true
- IsCygwinTerminal = false
- Color does work

Here's a sample of what it generally takes to track color support: https://github.com/efekarakus/termcolor and even this is error prone.

**Pull request merged**